server.ts

```diff
@@ -1,14 +1,74 @@
-import { serve } from "https://deno.land/std/http/server.ts";
+import { serve } from "https://deno.land/std/http/server.ts"
+import { serveFile } from "https://deno.land/std/http/file_server.ts"
 
-const DEFAULT_HTTP_PORT = 8080;
+// Bundling client
+const CLIENT_SOURCE_PATH = "client/app.ts"
+const DIST_PATH = "client/dist"
+const CLIENT_DIST_PATH = `${DIST_PATH}/bundle.js`
+const INDEX_SOURCE_PATH = "client/index.html"
+const INDEX_DIST_PATH = `${DIST_PATH}/index.html`
+const NOT_FOUND_SOURCE_PATH = "client/404.html"
+const NOT_FOUND_DIST_PATH = `${DIST_PATH}/404.html`
 
-const port = Number(Deno.args[0]) || DEFAULT_HTTP_PORT;
-const server = serve({ hostname: "0.0.0.0", port });
-console.log(`HTTP webserver running. Access it at: http://localhost:${port}/`);
+const [diagnostics, emit] = await Deno.bundle(CLIENT_SOURCE_PATH)
+await Deno.remove(DIST_PATH, { recursive: true })
+await Deno.mkdir(DIST_PATH)
+await Deno.writeTextFile(CLIENT_DIST_PATH, emit)
+await Deno.copyFile(INDEX_SOURCE_PATH, INDEX_DIST_PATH)
+await Deno.copyFile(NOT_FOUND_SOURCE_PATH, NOT_FOUND_DIST_PATH)
+
+// Running server
+const DEFAULT_HTTP_PORT = 8080
+
+const port = Number(Deno.args[0]) || DEFAULT_HTTP_PORT
+const server = serve({ hostname: "0.0.0.0", port })
+console.log(`HTTP webserver running. Access it at: http://localhost:${port}/`)
+
+async function fileExists(path: string) {
+  try {
+    const stats = await Deno.lstat(path)
+    return stats && stats.isFile
```

Comment:
Can we make that `fileExists` will always return `boolean` not `null` or `undefined`? I mean if you think stats can be `null` or `undefined` we still want to get `false` from `fileExists`.
Also looks (I am only checking typings here) like `Deno.lstat` can not return something we should check before using so maybe remove this check at all?